Sming/system/include/esp_systemapi.h

```diff
@@ -25,8 +25,11 @@
 #define STORE_TYPEDEF_ATTR __attribute__((aligned(4),packed))
 #define STORE_ATTR __attribute__((aligned(4)))
 
+extern int m_vsnprintf(char *buf, uint32_t maxLen, const char *fmt, va_list args);
+extern int m_printf(const char *fmt, ...);
+
 #undef assert
-#define debugf(fmt, ...) os_printf(fmt"\r\n", ##__VA_ARGS__)
+#define debugf(fmt, ...) /*m_printf(fmt"\r\n", ##__VA_ARGS__)*/
```

Comment:
Please provide more info in related thread: https://github.com/anakod/Sming/pull/169
But t shouldn't be here, of course.